test/integration/integration_suite_test.go

```diff
@@ -95,211 +64,84 @@ func TestIntegration(t *testing.T) {
 var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.UseDevMode(true), zap.WriteTo(GinkgoWriter)))
 
+	if v, ok := os.LookupEnv(dumpLogsEnvName); ok {
+		dumpLogs = strings.ToLower(v) == "true"
```

Comment:
```suggestion
		dumpLogs, _ = strconv.ParseBool(v)
```

This supports a few more truthy values like "T", "1", "True", etc.